### PR TITLE
Add Chinese translation

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_zh_CN.properties [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+AbstractFolder.BeingDeleted={0} \u51C6\u5907\u5220\u9664
+AbstractFolder.FailureToStopBuilds=Failed to interrupt and stop {0,choice,1#{0,number,integer} build|1<{0,number,\
+  integer} builds} of {1}
```
Comment (jglick) [inline src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_zh_CN.properties]: There should not be English messages here. If something is not yet translated, fine—just delete the entry.

File: src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_zh_CN.properties [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Folder.DisplayName=\u6587\u4EF6\u5939
+Folder.DefaultPronoun=Item
+Folder.Description=\u521B\u5EFA\u4E00\u4E2A\u53EF\u4EE5\u5D4C\u5957\u5B58\u50A8\u7684\u5BB9\u5668\u3002\u5229\u7528\u5B83\u53EF\u4EE5\u8FDB\u884C\u5206\u7EC4\u3002 \
+  \u89C6\u56FE\u4EC5\u4EC5\u662F\u4E00\u4E2A\u8FC7\u6EE4\u5668\uFF0C\u800C\u6587\u4EF6\u5939\u5219\u662F\u4E00\u4E2A\u72EC\u7ACB\u7684\u547D\u540D\u7A7A\u95F4\uFF0C \
+  \u56E0\u6B64\u4F60\u53EF\u4EE5\u6709\u591A\u4E2A\u76F8\u540C\u540D\u79F0\u7684\u7684\u5185\u5BB9\uFF0C\u53EA\u8981\u5B83\u4EEC\u5728\u4E0D\u540C\u7684\u6587\u4EF6 \
+  \u5939\u91CC\u5373\u53EF\u3002
+
+RelocateAction.permission.desc=Required to move a job from one folder (or Jenkins root) to another.
```
Comment (jglick) [inline src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_zh_CN.properties]: ditto

File: src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_zh_CN.properties [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+AbstractFolder.BeingDeleted={0} \u51C6\u5907\u5220\u9664
+AbstractFolder.FailureToStopBuilds=Failed to interrupt and stop {0,choice,1#{0,number,integer} build|1<{0,number,\
+  integer} builds} of {1}
+Folder.DisplayName=\u6587\u4EF6\u5939
+Folder.DefaultPronoun=Item
```
Comment (jglick) [inline src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_zh_CN.properties]: ditto

File: src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_zh_CN.properties (hunk 1 above)
Comment (LinuxSuRen) [reply src/main/resources/com/cloudbees/hudson/plugins/folder/Messages_zh_CN.properties]: thanks for your reminding

**Comment by LinuxSuRen** [comment]
already modify

**Pull request merged**